docker/release/README.md

```diff
@@ -0,0 +1,105 @@
+## OpenSearch/Dashboards Docker Build README
+
+### Summary
+We support building OpenSearch/Dashboards docker images with [multi-CPU architecture support](https://docs.docker.com/desktop/multi-arch/).
+We also support old school single CPU architecture build as well.
+Users are welcome to choose either type of the image to build on their local development environment, or directly pulling existing images published on Docker Hub:
+
+[OpenSearch Docker Repository](https://hub.docker.com/r/opensearchproject/opensearch/)
+
+[OpenSearch-Dashboards Docker Repository](https://hub.docker.com/r/opensearchproject/opensearch-dashboards/)
+
+```
+docker pull opensearchproject/opensearch:latest
+docker pull opensearchproject/opensearch-dashboards:latest
+```
+
+### Building Docker Images
+#### We provide two scripts to build docker images. For single-arch image you need to install just the Docker Engine on your host machine. For multi-arch image (currently support x64/arm64) you need to install Docker Desktop (macOS/Windows) or Docker Buildx (LINUX).
```

Comment:
Why is this a section title?

Reply:
This is needed as it separate build image, configure run, and actually running the docker container.